docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Edge multi-cluster"
+linkTitle: "Edge multi-cluster"
+weight: 1
+description: >
+  Investigation of edge multi-cluster
+---
+
+Kubernetes offers a state-based management platform to declaratively developing and operating workloads. Kube allows a developer to
+ create workloads that are node-agnostic and can be constrained to operate in places where certain conditions are met. Operators enjoy
+  gaurantee that workloads are rescheduled to work wherever constraints allow them to run. This approach has satisfied the large
+   majority of use cases in the cloud native community.
+
+There is a growing community of cloud native developers and operators that work for organizations that require more fine-grain control
+ over the distribution of workload due to the nature of their business. Telecommunications, industrial manufacturers, and automotive
+  companies would like to rely on Kubernetes state-based management but they need more flexibility to accomodate for conditions like
+   large-scale deployments, disconnected and autonomous operation, and constrained cluster size. These requirements represent the Edge 
+   Multi-cluster use case.
+
+A key area of investigation for `kcp` is exploring large-scale deployment of workloads that can operate under constraints and
+ conditions unlike those accomodated by upstream Kubernetes. Ideally, we want declaritively defined workloads to operate across a
+  hierarchy combining KCP workspaces and Kubernetes cluster namespaces to achieve greater scale and control over placement. Deployment
+   of workloads across many thousands of destinations that include namespaced and non-namespaced objects, receiving state and reporting
+    status (grouped and individually), require new interfaces based on (and extending) KCP primitives.
+
+## Goal: Placement strategies
+
+There are many different ways to characterize deployments of workloads in the Edge use case. Support for blue/green, canary, carbon
+ intensity sensitive, and threat avoidance are some of the deployment strategies to consider when casting out 100's or 1000's of
+  workloads to millions of destinations. Having the ability to assign workloads using a declaritively defined placement strategy to
+   edge cluster locations, even if they are offline at times, is a key requirement of the edge multi-cluster use case.
+
+## Overlap of roles, responsibilities, integrations, and security
+
```

Comment:
Can we have an example for something that is not covered by the existing KCP roles?

Reply:
can you give me an example of another case other than the smartphone analogy?

Reply:
I didn't think abut it so I don't have one (which is my here :-)  ). I thought the claim is that the `roles, responsibilities, integrations, and security`  of KCP doesn't address the Edge use case and need to be explored. If this is the claim I think you need to show an example of such a gap

Reply:
I am not sure "Overlap" is the right focus here.  In many ways, the point is about _divisions_ rather than _overlap_.  I usually say this with words like "a more complex arrangement of roles and responsibilities".  I am not sure "integrations" and "security" belong in the same breath.

TMC does not have something like the arrangement of roles and responsibilities that Andy described for smartphones.  Other examples I use:
- personal computer issued to corporate employee
- car with software managed by owner, manufacturer, possibly dealer, possibly other service provider
- a manufacturing operation with many plants with local authority as well as a corporate engineering team and a web of suppliers

Reply:
I removed the word 'integrations'